### PR TITLE
Minor Everything Bagel improvements

### PR DESCRIPTION
**Pull request by rmoff**
- Remove password auth to make DX smoother
- Add some sample data
- Add Spark SQL magic support
- Bump notebook base version
- Install lakeFS python client

### CODE REVIEW

File: deployments/compose/jupyter/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+lakefs_client
+jupysql==0.5.2
+# matplotlib==3.6.3
```
Comment (arielshaqed) [inline deployments/compose/jupyter/requirements.txt:3]: Usually we prefer no code in comments. I understand why these are here, though - so let's just make sure these are compatible and loadable together into the same Python interpreter instance.

**Review by arielshaqed — APPROVED** [review]
Neat... Thanks!
How was this tested? (Standard review question, "by running xyzzy" is a great answer)

File: deployments/compose/jupyter/requirements.txt (hunk 1 above)
Comment (rmoff) [reply deployments/compose/jupyter/requirements.txt:3]: thanks - removed them

**Comment by rmoff** [comment]
> Neat... Thanks! How was this tested? (Standard review question, "by running xyzzy" is a great answer)

Thanks @arielshaqed. Tested manually with

```
docker-compose build && docker-compose up notebook lakefs
```

and then running a notebook.

**Pull request merged**